app/forms/forms/submission_email_form.rb

```diff
@@ -0,0 +1,94 @@
+class Forms::SubmissionEmailForm
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :form, :temporary_submission_email, :email_code, :confirmation_code, :current_user
+
+  EMAIL_REGEX = /.*@.*/
+  GOVUK_EMAIL_REGEX = /\.gov\.uk\z/i
+
+  validates :temporary_submission_email, presence: true
+  validates :temporary_submission_email, format: { with: EMAIL_REGEX, message: :invalid_email }, if: -> { temporary_submission_email.present? }
+  validates :temporary_submission_email, format: { with: GOVUK_EMAIL_REGEX, message: :non_govuk_email }, if: -> { temporary_submission_email.present? }
```

Comment:
Not sure these validations make sense together - think the `presence: true` validation means that the  `if: -> { temporary_submission_email.present? }` in the other two doesn't do anything. That condition makes sense in the existing email form because there it's sometimes valid to have a blank email.

Reply:
Good spot! I've removed them